packages/cli/src/constructs/alert-channel.ts

```diff
@@ -54,11 +66,15 @@ export abstract class AlertChannel extends Construct {
     this.sslExpiryThreshold = props.sslExpiryThreshold
   }
 
+  static fromId (id: string|number) {
+    return new AlertChannelWrapper(`alert-channel-${id}`, id)
+  }
+
   allowInChecklyConfig () {
     return true
   }
 
-  synthesize () {
+  synthesize (): any|null {
```

Comment:
```suggestion
  synthesize (): {
    sendRecovery?: boolean
    sendFailure?: boolean
    sendDegraded?: boolean
    sslExpiry?: boolean
    sslExpiryThreshold?: number
    } {
```

Reply:
We should do this but not with this PR IMO